pages/how-to-create-module.vue

```diff
@@ -81,7 +81,21 @@
       </li>
       <li>
         Define the libraries in your module by adding one call to
-        <code>add_geode_library</code> per library.
+        <code>add_geode_library</code> per library. Here are the basic parameters (for advanced parameters, see examples in OpenGeode code):
+        <ul>
+          <li>
+            <code>NAME</code> the name of the library
+          </li>
+          <li>
+            <code>FOLDER</code> the folder hierarchy where your files are located
```

Comment:
Maybe you can be more explicit saying that is the folder name from project_name/include and project_name/src ?